lib/value/src/value/lua.rs

```diff
@@ -107,7 +107,10 @@ pub fn table_to_timestamp(t: LuaTable<'_>) -> LuaResult<DateTime<Utc>> {
     let min = t.raw_get("min")?;
     let sec = t.raw_get("sec")?;
     let nano = t.raw_get::<_, Option<u32>>("nanosec")?.unwrap_or(0);
-    Ok(Utc.ymd(year, month, day).and_hms_nano(hour, min, sec, nano))
+    Ok(Utc
+        .ymd(year, month, day)
+        .and_hms_opt(hour, min, sec)
+        .expect("invalid datetime"))
```

Comment:
This does seem like a breaking change. It looks like there is a [and_hms_nano_opt](https://docs.rs/chrono/latest/chrono/naive/struct.NaiveDate.html#method.and_hms_nano_opt) ?

Reply:
Yeah I did mention it in the description as a minor breaking change. I missed `and_hms_nano_opt` though, good find.